x-pack/plugins/stack_alerts/server/rule_types/es_query/executor.ts

```diff
@@ -148,13 +147,13 @@ export async function executor(core: CoreSetup, options: ExecutorOptions<EsQuery
       actionGroup: ActionGroupId,
       state: { latestTimestamp, dateStart, dateEnd },
       context: actionContext,
-      payload: expandFlattenedAlert({
+      payload: {
         [ALERT_URL]: actionContext.link,
         [ALERT_REASON]: actionContext.message,
         [ALERT_TITLE]: actionContext.title,
         [ALERT_EVALUATION_CONDITIONS]: actionContext.conditions,
-        [ALERT_EVALUATION_VALUE]: actionContext.value,
-      }),
+        [ALERT_EVALUATION_VALUE]: `${actionContext.value}`,
```

Comment:
**question:** what caused string to be the required type for es_query and index_threshold? Is it a float which works mainly with strings to support long values?

Reply:
This is because when the field map was defined for the evaluation value, it specified `keyword` mapping type and in our schema generation script, it converts `keyword` to string schema. I actually think the mapping should have been numerical, it was an oversight :(

However, I think O11y rules are using `ALERT_EVALUATION_VALUES` (plural not singular) so we should probably update to using that as well and stop populating the singular version.